### ISSUE TITLE
Column Block: Accessibility Improvement Request - Label each column so that a user can know which column they are editing

### ISSUE THREAD

**Issue by dcoleonline** [Automattic/wp-calypso#44269]
#### Steps to reproduce

1. Start with a new post or a page
2. Add a columns block and add some content to either editor and save the content
3. Use a screen reader to try to navigate the editor. (I used Voiceover on an iPhone 7)
4. When you navigate between blocks, there is no mention of the fact that you are editing within a columns block or which column you are editing.

#### What I expected
To be told which column I was editing

#### What happened instead
The columns were not announced via the screen reader software

#### Browser / OS version
iPhone 7 using Voiceover on 13.5.1

#### Screenshot / Video
Screenshots of what the editor looks like on desktop:
![image](https://user-images.githubusercontent.com/3505172/87858184-aa6a7b00-c8f1-11ea-9591-ca57ec4c1d36.png)
![image](https://user-images.githubusercontent.com/3505172/87858187-b48c7980-c8f1-11ea-945a-6560d51bead9.png)

Screencast of using Voiceover on iOS to try to navigate between columns.
https://cloudup.com/c2dk3oWzf2H

#### Context / Source
This was discovered while assisting a visually impaired user in 3030290-zen

> I created a 2 column block and in 1 block wanted to create a pay pal block and in the second block a payment reoccurring block and the column block was created but when I swiped right to find the first block my voice over said ad block, block pop pup button, add block, add block... so it was confusing. If possible it would be great if say a 2 column bloc was created it would be labeled to say block 1/block 2 or column 1/column 2 so they are identified.
